### ISSUE TITLE
retweet was not recognised

### ISSUE THREAD

**Issue by Kikobeats** [omrilotan/isbot#232]
User Agent String
-------

```
retweet/2.17 (iPhone; iOS 17.1.2; Scale/3.00)
retweet/2.17 (iPhone; iOS 17.2.1; Scale/3.00)
retweet/2.17 (iPhone; iOS 17.1.2; Scale/2.00)
```

Reproduce
-------

I belive it comes from an iPhone app.

**Comment by omrilotan** [comment]
Please see https://github.com/omrilotan/isbot/pull/233

**Comment by omrilotan** [comment]
Resolved in [4.1.1](https://github.com/omrilotan/isbot/blob/next/CHANGELOG.md#411).
Updating from version 3? Please see [migration instructions](https://github.com/omrilotan/isbot/blob/next/CHANGELOG.md#411)
